.gitignore

```diff
@@ -52,9 +52,12 @@ qtcreator-*
 *.swp
 
 # Catkin custom files
-CATKIN_IGNORE
+# CATKIN_IGNORE
```

Comment:
Something very meta about ignoring (commenting) the git ignoring of the catkin ignore file here.

Maybe just delete this line?

Reply:
agree, I remove it.